code/game/objects/items/RCD.dm

```diff
@@ -203,10 +203,23 @@ RLD
 	var/use_one_access = 0 //If the airlock should require ALL or only ONE of the listed accesses.
 	var/delay_mod = 1
 	var/canRturf = FALSE //Variable for R walls to deconstruct them
+	var/id
+	var/obj/machinery/conveyor/last_placed
 
 /obj/item/construction/rcd/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] sets the RCD to 'Wall' and points it down [user.p_their()] throat! It looks like [user.p_theyre()] trying to commit suicide..</span>")
 	return (BRUTELOSS)
+	
+/obj/item/construction/rcd/attackby(obj/item/I, mob/user, params)
+	..()
+	if(upgrade & RCD_UPGRADE_CONVEYORS)
+		if(istype(I, /obj/item/stack/conveyor))
+			loadwithsheets(I, 12, user)
+			to_chat(user, "<span class='notice'>[src] now holds [matter]/[max_matter] matter-units.</span>")
```

Comment:
Remove this bit, feeding the RCD conveyors is silly.

Reply:
I suppose, but when I added it I thought it would be neat to be able to deconstruct and reload the RCD on the go, without the need of running to the lathe and back to recycle the conveyors. I can remove it if it's not a good idea.